Apromore-Core-Components/Apromore-Portal/src/main/webapp/themes/ap/common/js/upload.js

```diff
@@ -51,21 +77,33 @@ Ap.encodingDetect = function () {
                             return a
                         },
                         {}
-                    )
-
-                enc = enc.toLowerCase()
-                let option = options[enc]
+                    );
+                enc = enc.toLowerCase();
+                if (enc === 'ascii') {
+                    enc = 'utf-8';
+                }
+                let option = options[enc];
                 if (option) {
+                    success = true;
                     // Do it via proxy
-                    // console.log('fire', option.index)
                     setTimeout(function () {
+                        Ap.common.notify('Encoding is detected as ' + option.value, 'info');
                         wgt.setValue("" + option.index);
-                        wgt.fireOnChange({ toServer: true })
+                        wgt.fireOnChange({ toServer: true });
                     }, 200)
```

Comment:
If you put a "return;" right here, could you eliminate the "success" flag and simplify the logic considerably?